### ISSUE TITLE
Arrow navigation is not implemented for the cards.

### ISSUE THREAD

**Issue by 232amansingh** [microsoft/BotFramework-WebChat#5743]
### Is it an issue related to Adaptive Cards?

Yes, this is related to Adaptive Card. However, it does not repro on Adaptive Cards Designer.

### What is the PWD impact?

When arrow key navigation is not implemented for cards, keyboard users cannot move between cards efficiently. They may be forced to tab through the entire page, which is slow and confusing, especially if each card contains multiple interactive elements.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Repro Steps: ​           
1. Open URL: Web Chat in edge browser.
2. Ask query 'Carousel'.
3. Navigate to cards.
4. Observe the issue that Arrow navigation is not implemented for the cards.

### What do you expect?

Arrow navigation should be implemented for the cards.

### What actually happened?

Arrow navigation is not implemented for the cards.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/96d7cb90-db16-4902-81f3-e0488f3e6591

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/keyboard

### Adaptive Card JSON

```json

```
